src/wizards/generateProject/generationWizard.ts

```diff
@@ -58,14 +58,35 @@ export async function generateProjectWizard() {
     });
 
     state.buildTool = (await input.showQuickPick({
-      title: 'Quarkus Tools',
+      title: INPUT_TITLE,
       step: input.getStepNumber(),
       totalSteps: state.totalSteps,
       placeholder: 'Pick build tool',
       items: quickPickItems,
       activeItem: quickPickItems[0]
     })).label;
 
+    return (input: MultiStepInput) => inputPlatformVersion(input, state);
+  }
+
+  async function inputPlatformVersion(input: MultiStepInput, state: Partial<ProjectGenState>) {
+
+    const quickPickItems: PlatformVersionPickItem[] = await getCodeQuarkusApiPlatforms();
+
+    // Place the recommended option first in the quick pick list
+    quickPickItems.sort((x: PlatformVersionPickItem, y: PlatformVersionPickItem) => {
+      return (x.recommended === y.recommended) ? 0 : x.recommended ? -1 : 1;
+    });
+
+    state.platformVersion = (await input.showQuickPick({
+      title: INPUT_TITLE,
+      step: input.getStepNumber(),
+      totalSteps: state.totalSteps,
+      placeholder: 'Pick io.quarkus.platform',
```

Comment:
Minor nitpicks :

- "Pick io.quarkus.platform version"
- If we're going to use "io.quarkus.platform" in the instruction/hint, then there's no need to include it also in each item. I would just have each item label show the version (eg. 2.2, 2.1) and a label like "2.2 (recommended)" where recommended is true.